### ISSUE TITLE
SOLUTION TO MULTIPLE ERRORS!

### ISSUE THREAD

**Issue by SilentDelirium** [charlesbel/Microsoft-Rewards-Farmer#404]
I found a lot of people on here posting different errors they were receiving when running the script. I am pretty sure I have had all of them at least once.

I found that in the main folder containing the script was a folder called "sessions". This folder was HUGE (2.5-3GB), and I couldn't delete it. It always stated that the files were in use, even after killing the script and all instances of Chrome...

So, I got this program I used to use back in the day (I hear it was a Thursday) called Unlocker. It's old freeware that tries to - well, unlock files so that you can rename/move/delete them. It usually works wonders, but even that couldn't delete the sessions folder... BUT, when it failed to rename the folder (i thought it might be easier than deleting outright), the program asked if I wanted to try the operation at boot. I hit yes, and rebooted. Bob's your mother's brother!

The script works great!

I am still pretty new to Github, and only just learned python, but if someone benefits from this and knows what the hell they're doing, could you inform the dev of the issue? Or maybe "sticky" this to the issues or posts?

**Comment by SilentDelirium** [comment]
Am I seriously writing a solution as error number 404!! Ha!

**Comment by klept0** [comment]
Use the kill script in previous posts to completely kill Chrome and you can edit/delete the folder without having to use a 3rd party app

**Comment by SilentDelirium** [comment]
True! Good call. Though, I did a kill cmd to chrome, abd that didn't work for me personally. Not sure if it's a Google drive sync issue, or something random like that.

**Comment by ebuwiz** [comment]
Using the script on MacOS Sonoma... Using Chromium 119.0.6025.0, disabled updates, 0 errors... On windows i remember i have to disable Google Updates or script comes unstable and unrunnable

**Comment by alessiouw** [comment]
Resolved my issues! Deeting the folder sessions (it was like 21GBs for me) solved it.

**Comment by cyberpods** [comment]
Wow this is also what was causing my issues. Thanks
